lib/nebulex/adapters/replicated.ex

```diff
@@ -266,7 +266,7 @@ defmodule Nebulex.Adapters.Replicated do
   # Inherit default persistence implementation
   use Nebulex.Adapter.Persistence
 
-  use Bitwise, only_operators: true
+  import Bitwise, only_operators: true
```

Comment:
If we replace `use` with `import`, which is ok, you have to use the proper `import` options, because `only_operators` works only with the `use`. Check it out: https://github.com/elixir-lang/elixir/blob/main/lib/elixir/lib/bitwise.ex#L26-L42

So, I suggest using the `import` with the `only` option, like `use Bitwise, only: [<<<: 2]` (I think that is the only operator the adapter is using)

Reply:
@cabol oh right! apologies I didnt noticed it, I added another commit https://github.com/cabol/nebulex/pull/172/commits/8b3b5962382465a2dec50c06f60a09869def974e to do `import Bitwise, only: [<<<: 2]`, let me know what you think

Reply:
Looks 👌 !!